frontend/src/app/modules/projects/project-list/project-list.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+    selector: 'app-project-list',
+    templateUrl: './project-list.component.html',
+    styleUrls: ['./project-list.component.sass'],
+})
+
+export class ProjectListComponent {
+    @Input() project: any;
```

Comment:
Can you create model instead any and place name place to "project-card" or something similar?